paddle/fluid/platform/mkldnn_helper.h

```diff
@@ -328,6 +328,23 @@ inline mkldnn::memory::format_tag GetMKLDNNFormat(
   return mkldnn::memory::format_tag::undef;
 }
 
+inline mkldnn::memory::format_tag GetPlainMKLDNNFormat(int tensor_rank) {
```

Comment:
Just a note since I have a feeling that I saw this kind of code multiple times in many PR's. Please make sure that it is reused anywhere it could be.

Reply:
Also, is 6 dimensions biggest dimension it could be, because there used to be some issues caused by unsupported format.